src/main/java/dev/openfga/sdk/api/model/ConsistencyPreference.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * OpenFGA
+ * A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar.
+ *
+ * The version of the OpenAPI document: 1.x
+ * Contact: community@openfga.dev
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package dev.openfga.sdk.api.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * - UNSPECIFIED: Default if not set. Behavior will be the same as MINIMIZE_LATENCY  - MINIMIZE_LATENCY: Minimize latency at the potential expense of lower consistency.  - HIGHER_CONSISTENCY: Prefer higher consistency, at the potential expense of increased latency.
+ */
+public enum ConsistencyPreference {
+    UNSPECIFIED("UNSPECIFIED"),
+
+    MINIMIZE_LATENCY("MINIMIZE_LATENCY"),
+
+    HIGHER_CONSISTENCY("HIGHER_CONSISTENCY"),
+
+    UNKNOWN_DEFAULT_OPEN_API("unknown_default_open_api");
+
+    private String value;
+
+    ConsistencyPreference(String value) {
+        this.value = value;
+    }
+
+    @JsonValue
+    public String getValue() {
+        return value;
+    }
+
+    @Override
+    public String toString() {
+        return String.valueOf(value);
+    }
+
+    @JsonCreator
+    public static ConsistencyPreference fromValue(String value) {
+        for (ConsistencyPreference b : ConsistencyPreference.values()) {
+            if (b.value.equals(value)) {
+                return b;
+            }
+        }
+        return UNKNOWN_DEFAULT_OPEN_API;
```

Comment:
Is there a way to indicate UNSPECIFIED is the default?

Reply:
the generated javadocs for the class state that UNSPECIFIED is the default. I'm not how to hook into the javadocs generation for more customization but I'm guessing it's possible